modules/rgbd/src/tsdf.hpp

```diff
@@ -47,12 +50,14 @@ class TSDFVolume : public Volume
     Vec8i neighbourCoords;
 };
 
+cv::Mat preCalculationPixNorm(Depth depth, const Intr& intrinsics);
```

Comment:
Please make this function static and do not put its declaration into the header

Reply:
done